integration-tests/src/test_loop/tests/resharding_v3.rs

```diff
@@ -764,23 +772,25 @@ fn test_resharding_v3_base(params: TestReshardingParameters) {
         // Give enough time to produce ~7 epochs.
         Duration::seconds((7 * params.epoch_length) as i64),
     );
-    let client = &env.test_loop.data.get(&client_handles[0]).client;
+    let client = &env.test_loop.data.get(&client_handles[client_index]).client;
     trie_sanity_check.check_epochs(client);
     let height_after_resharding = latest_block_height.get();
 
     // Delete `temporary_account`.
-    delete_account(&mut env, &rpc_id, &temporary_account, &rpc_id);
+    delete_account(&mut env, &client_account_id, &temporary_account, &client_account_id);
     // Wait for garbage collection to kick in.
     env.test_loop
         .run_for(Duration::seconds((DEFAULT_GC_NUM_EPOCHS_TO_KEEP * params.epoch_length) as i64));
-    // Check that the deleted account is still accessible at archival node, but not at a regular node.
-    check_deleted_account_availability(
-        &mut env,
-        &archival_id,
-        &rpc_id,
-        temporary_account,
-        height_after_resharding,
-    );
+    if let Some(archival_id) = archival_id {
```

Comment:
Is it ok to skip this check if there is no archival node?

Reply:
I've changed `check_deleted_account_availability` to make the check on the RPC node in any case and to make the check on the archival only if present.